packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.tsx

```diff
@@ -124,6 +126,28 @@ export default function CreateTableViewWizard(
       },
     })
 
+  const {
+    mutateAsync: validateSql,
+    data: isSqlValid,
+    error: sqlValidationError,
+  } = useValidateDefiningSql(sql, entityType!, {
+    onSuccess: (isSqlValid: ValidateDefiningSqlResponse) => {
+      if (isSqlValid && isSqlValid.isValid) {
+        setStep('TABLE_NAME')
+      }
+    },
+  })
+
+  const showSqlValidationError: string | null = useMemo(() => {
+    if (sqlValidationError?.message) {
+      return sqlValidationError.message
```

Comment:
Is this necessary? I think we set up the generics so `useValidateDefiningSql`'s error will always be `SynapseClientError` if it is defined